navigation/client/navigationLinks.js

```diff
@@ -29,3 +25,10 @@ Template._navigationLinks.helpers({
     showOrHideExportCSVButton: function() {
       return isPage('responses') ? '' : 'hide'; }
 });
+
+Template._navigationLinks.events({
+    "click .share-form": function () {
+        // Tell Sandstorm to show the Share Access menu.
+        window.parent.postMessage({startSharing: {}}, "*");
```

Comment:
I vaguely suggest using `true` not `{}` if possible.

Presumably someone should write some docs for this in the Sandstorm codebase, too. Maybe that's on me. (-: